modules/t/test-genome-DBs/mus_musculus/variation/meta.txt

```diff
@@ -99,3 +99,4 @@
 105	\N	patch	patch_110_111_a.sql|schema version
 106	\N	patch	patch_110_111_b.sql|Update transcript_variation primary key
 107	\N	patch	patch_111_112_a.sql|schema_version
+108	\N	patch	patch_112_113_a.sql|schema_version
```

Comment:
Good point and I agree we should have a b (and also c when it's incorporated shortly) patch. I believe it's because no such patch exists in the ensembl-variation repo, and it seems the b and c patches should be incorporated into that repo so they're picked up here.